apps/parsley/package.json

```
@@ -19,6 +19,7 @@
     "deploy:prod": "env-cmd -e production ts-node scripts/deploy/run-deploy",
     "deploy:do-not-use": ". ./scripts/deploy/deploy.sh",
     "dev": "env-cmd -e devLocal -r .env-cmdrc.local.json vite",
+    "evg-db-ops": "../../scripts/evg-db-ops.sh",
```

since you added `evg-db-ops` to the `main package.json`, I wonder if it works if you do
```suggestion
    "evg-db-ops": "yarn --cwd='../..' evg-db-ops",
```
Sophie did something similar for the `prepare` command